lib/android/README.md

```diff
@@ -6,6 +6,8 @@
 
 Android SDK and NDK for Linux x86\_64. The Android SDK is expected to be installed at `~/Android/Sdk`. If it is somewhere else instead, you can specify it with a Make variable, e.g. `make ANDROID_SDK_ROOT=/other/location/android-sdk`
 
+DISCLAIMER: NDK 23.0.7123448 removed some of the tools needed for building this target, as we can't assure they will be added again the NDK 22.0.7026061 is assured to work.
```

Comment:
It's probably because it's a beta release that not all the prebuilt tools are included, so I suggest pointing that out instead of saying they've been removed.

[I found one issue on `ring` tracking this as well.](https://github.com/briansmith/ring/issues/1206)

```suggestion
DISCLAIMER: NDK 23 is currently in beta, so it might not have all the prebuilt tools used in the build process, please stick to using a stable version, for example NDK 22.
```